web-components/src/components/favorite/Favorite.ts

```diff
@@ -39,11 +39,10 @@ export namespace Favorite {
     }
 
     getIconName() {
-      switch (this.checked) {
-        case true:
-          return "favorite-active_16";
-        default:
-          return "favorite_16";
+      if (this.checked) {
```

Comment:
use a shorthand: `return this.checked ? "favorite-active_16" :  "favorite_16";`

Reply:
Done